src/main/java/io/temporal/samples/dsl/models/Statement.java

```diff
@@ -0,0 +1,37 @@
+package io.temporal.samples.dsl.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Map;
+
+public class Statement {
+  public ActivityInvocation activity;
+  public Sequence sequence;
+  public Parrallel parrallel;
```

Comment:
@mfateev You made a comment about using inheritance.  Can you give me an example of what you mean?  I wasn't following what can inherit from what?

Reply:
interface Statement {}
class Sequence implements Statement {}
class ActivityInvocation implements Statement {}
class Parallel implements Statement {}

Reply:
I'm not seeing what the interface could be.  Here is wha we have?
```
class Statement {
   public ActivityInvocation activity;
   public Sequence sequence;
   public Parallel parallel;
}

class Sequence {
  public Statement elements[];
}

class Parallel  {
  public Statement branches[];
}

class ActivityInvocation {
  public String name;
  public String method;
  public String[] arguments;
  public String result;
}
```

The only thing similar is `sequence` and `parallel`, but you would have to change the name to be consistent `elements` vs. `branches`.

Reply:
@mfateev ping